### ISSUE TITLE
Cannot analyse Gradle parallell build

### ISSUE THREAD

**Issue by pkubowicz** [snyk/snyk#409]
- `node -v`: v8.15.1
- `npm -v`: 6.4.1
- `snyk -v`: 1.143.1
- Command run: snyk test --org=my-organisation --project-name=test

### Expected behaviour
Snyk performs analysis

### Actual behaviour
```
Testing /home/piotr-kubowicz/tmp/bugs/snyk-parallel-gradle...

More than one line with JSONDEPS prefix was returned

Please make sure that `/home/piotr-kubowicz/tmp/bugs/snyk-parallel-gradle/gradlew snykResolvedDepsJson -q --build-file build.gradle --no-daemon -I .../node_modules/snyk-gradle-plugin/lib/init.gradle` executes successfully on this project.

If the problem persists, collect the output of `/home/piotr-kubowicz/tmp/bugs/snyk-parallel-gradle/gradlew snykResolvedDepsJson -q --build-file build.gradle --no-daemon -I .../node_modules/snyk-gradle-plugin/lib/init.gradle` and contact support@snyk.io
```

```
JSONDEPS {"$defaultProject":"frontend"}
JSONDEPS {"$defaultProject":"snyk-parallel-gradle","snyk-parallel-gradle":{"io.swagger:swagger-annotations":{"name":"io.swagger:swagger-annotations","version":"1.5.13"}}}
```

### Steps to reproduce
Run Snyk on any multi-project Gradle build with parallel build enabled. For example on https://github.com/pkubowicz/snyk-parallel-gradle

**Comment by kyegupov** [comment]
@pkubowicz we are aware of the problem and will deploy the fix soon. Meanwhile, you can work around by adding a gradle flag:

`snyk test --org=my-organisation --project-name=test -- -no-parallel`

**Comment by lili2311** [comment]
We have released a version 1.143.2 of the snyk cli that should help with your issue. Please give it a try and let me know if the issue still occurs for you :)

**Comment by pkubowicz** [comment]
Works fine for me.
